src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberMethodSymbol.cs

```diff
@@ -23,11 +24,11 @@ protected struct Flags
         {
             // We currently pack everything into a 32 bit int with the following layout:
             //
-            // |   |s|r|q|z|xxxxxxxxxxxxxxxxxxxxxxx|wwwww|
+            // | |t|s|r|q|z|xxxxxxxxxxxxxxxxxxxxxxxx|wwwww|
             // 
             // w = method kind.  5 bits.
             //
-            // x = modifiers.  23 bits.
+            // x = modifiers.  24 bits.
```

Comment:
>// x = modifiers.  24 bits. [](start = 12, length = 27)

What is the motivation for the change? It looks like DeclarationModifiers can fit into 23 bits ([https://github.com/dotnet/roslyn/blob/master/src/Compilers/CSharp/Portable/Declarations/DeclarationModifiers.cs](https://github.com/dotnet/roslyn/blob/master/src/Compilers/CSharp/Portable/Declarations/DeclarationModifiers.cs)).